### PR TITLE
load after rsce

### PR DESCRIPTION
**Pull request by fritzmg**
I use both RSCE and the StyleSelector in almost all my projects - however one thing I always have to accommodate for is that the style selector is not available in the RockSolid Custom Elements. This simple change makes them available :)

_Note:_ this will __not__ create a hard dependency on RSCE. If the RSCE bundle is not present, nothing bad will happen ;)

### CODE REVIEW

**Comment by iCodr8** [comment]
What happens with environments that not use the RSCE. I think there will be thrown an exception. Can you add the RSCE support only optional?

**Comment by fritzmg** [comment]
As already said, this does not create a hard dependency. No RSCE class will be actually loaded.

File: src/ContaoManager/Plugin.php [hunk 1]
```diff
@@ -11,9 +11,12 @@
 
 namespace Craffft\CssStyleSelectorBundle\ContaoManager;
 
-use Contao\ManagerPlugin\Bundle\Config\BundleConfig;
+use Contao\CoreBundle\ContaoCoreBundle;
 use Contao\ManagerPlugin\Bundle\BundlePluginInterface;
+use Contao\ManagerPlugin\Bundle\Config\BundleConfig;
 use Contao\ManagerPlugin\Bundle\Parser\ParserInterface;
+use Craffft\CssStyleSelectorBundle\CraffftCssStyleSelectorBundle;
+use MadeYourDay\RockSolidCustomElements\RockSolidCustomElementsBundle;
```
Comment (iCodr8) [inline src/ContaoManager/Plugin.php]: When this dependency is not existing, then an exception is thrown.

File: src/ContaoManager/Plugin.php [hunk 2]
```diff
@@ -28,8 +31,8 @@ class Plugin implements BundlePluginInterface
     public function getBundles(ParserInterface $parser)
     {
         return [
-            BundleConfig::create('Craffft\CssStyleSelectorBundle\CraffftCssStyleSelectorBundle')
-                ->setLoadAfter(['Contao\CoreBundle\ContaoCoreBundle'])
+            BundleConfig::create(CraffftCssStyleSelectorBundle::class)
+                ->setLoadAfter([ContaoCoreBundle::class, RockSolidCustomElementsBundle::class])
```
Comment (iCodr8) [inline src/ContaoManager/Plugin.php]: And here the same.

File: src/ContaoManager/Plugin.php (hunk 1 above)
Comment (fritzmg) [reply src/ContaoManager/Plugin.php]: Did you test that? In my tests it didn't. Use statements do not try to load the class.

File: src/ContaoManager/Plugin.php (hunk 2 above)
Comment (fritzmg) [reply src/ContaoManager/Plugin.php]: ::class statements do not need the class to be loaded

**Comment by iCodr8** [comment]
Everything seems okay. Thank you :)

**Pull request merged**